test/test-cleanFilledRows.js

```diff
@@ -64,5 +67,68 @@ describe("Check situation on field", function () {
             --X
             -X-
         `)
+        assert.deepEqual(resultCheckField, expectedResult)
+    })
+    it("Remove lines and add L tetromino", function () {
+        let fieldSize = tetris.buildField(`
+            X--
+            XXX
+            --X
+            XXX
+            -X-
+        `)
+        let coordinatesTetromino_L = tetris.getCoordinates(`
+            XXX
+            X--
+        `)
+        let field = new tetris.Field(fieldSize)
+        let firstResultCheckField = field.cleanFilledRows()
+        let addTetromino = field.addTetrominoToField(coordinatesTetromino_L)
+        let secondResultCheckField = field.cleanFilledRows()
+        let expectedResult = tetris.buildField(`
+            ---
+            X--
+            X--
+            --X
+            -X-
+        `)
+        assert.deepEqual(secondResultCheckField, expectedResult)
+    })
+    it("Cycle remove lines and add tetrominos", function () {
+        let fieldSize = tetris.buildField(`
+            X--
+            XXX
+            --X
+            XXX
+            -X-
+        `)
+        let coordinatesTetromino_L = tetris.getCoordinates(`
+            XXX
+            X--
+        `)
+        let coordinatesTetromino_Z = tetris.getCoordinates(`
+            XX-
+            -XX
+        `)
+        let coordinatesTetromino_J = tetris.getCoordinates(`
+            XXX
+            --X
+        `)
+        let field = new tetris.Field(fieldSize)
+        let firstResultCheckField = field.cleanFilledRows()
+        let addTetromino = field.addTetrominoToField(coordinatesTetromino_L)
+        let secondResultCheckField = field.cleanFilledRows()
+        let addNewTetromino = field.addTetrominoToField(coordinatesTetromino_Z)
+        let thirdResultCheckField = field.cleanFilledRows()
+        let addNewTetrominoJ = field.addTetrominoToField(coordinatesTetromino_J)
+        let forthResultCheckField = field.cleanFilledRows()
+        let expectedResult = tetris.buildField(`
```

Comment:
Add intermediate asserts for state of the field

Reply:
some assert.deepEqual in one "it" do not work